introduction.md

```diff
@@ -1,2 +1,8 @@
 Hi I am Glenn.
 My favorite Code School path is JavaScript.
+Deadly skills
+* Git
+* JavaScript
+* C++
+* Java
+* CSS
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?